app/views/questions/show.html.haml

```diff
@@ -0,0 +1,3 @@
+%h1 Add a New Question
+%div.question-form-div= render 'form'
```

Comment:
`%div.question-form-div` can be written as `.question-form-div` since `%div` is implicit